playbooks/setup_private_ai_stack.yml

```diff
@@ -88,11 +88,11 @@
         key: "{{ item }}"
       loop: "{{ additional_public_keys | default([]) }}"
 
-    - name: Registration sle-micro
+    - name: Registration SUSE OS
       become: true
       shell: |
         transactional-update register --url {{ registration_server | default("https://scc.suse.com") }} --email {{ registration_email }} --regcode "{{ registration_code }}"
-      when: (cluster is defined and cluster.image_distro == "sle-micro") or (cluster is not defined and '"openSUSE" not in cloud_image_download_url')
+      when: '"openSUSE" not in ansible_facts.distribution'
       register: register_result
```

Comment:
when: ansible_distribution == 'SL-Micro' should be sufficient here..

Reply:
Thanks. I'll update.